### PR TITLE
Pact contracts should be tagged in CI

### PR DESCRIPTION
**Pull request by fareedmohammed**
https://eaflood.atlassian.net/browse/PSD-792

### CODE REVIEW

File: src/uk/gov/defra/ffc/GitHubStatus.groovy [hunk 1]
```diff
@@ -52,6 +52,11 @@ class GitHubStatus implements Serializable {
     static final Description = 'Snyk test'
   }
 
+  class PactBrokerTest {
```
Comment (johnwatson484) [inline src/uk/gov/defra/ffc/GitHubStatus.groovy:55]: I would rename this to something like `PublishPact` as it's not really running a test, it's publishing any contracts to the broker.

File: src/uk/gov/defra/ffc/GitHubStatus.groovy [hunk 2]
```diff
@@ -52,6 +52,11 @@ class GitHubStatus implements Serializable {
...
+    static final Context = "${BASECTX}pact-broker-test"
```
Comment (johnwatson484) [inline src/uk/gov/defra/ffc/GitHubStatus.groovy:56]: As above, suggest rename

File: src/uk/gov/defra/ffc/GitHubStatus.groovy [hunk 3]
```diff
@@ -52,6 +52,11 @@ class GitHubStatus implements Serializable {
...
+    static final Description = 'PactBroker test'
```
Comment (johnwatson484) [inline src/uk/gov/defra/ffc/GitHubStatus.groovy:57]: As above, suggest rename

File: vars/buildNodeJs.groovy [hunk 4]
```diff
@@ -80,8 +80,10 @@ void call(Map config=[:]) {
           utils.replaceInFile(containerSrcFolder, localSrcFolder, lcovFile)
         }
 
-        stage('Publish pact broker') {
-          pact.publishContractsToPactBroker(repoName, version.getPackageJsonVersion(), utils.getCommitSha())
+        if (pr == '') {
+          stage('Publish pact broker') {
+            pact.publishContractsToPactBroker(repoName, version.getPackageJsonVersion(), utils.getCommitSha(), defaultBranch)
```
Comment (johnwatson484) [inline vars/buildNodeJs.groovy:85]: The `defaultBranch` parameter isn't used so can be removed.

**Review by johnwatson484 — COMMENTED** [review]
Looks good.  A suggestion over naming and there's a parameter not used but other than that seems exactly what we need.

File: src/uk/gov/defra/ffc/GitHubStatus.groovy (hunk 1 above)
Comment (fareedmohammed) [reply src/uk/gov/defra/ffc/GitHubStatus.groovy:55]: Agreed.

File: src/uk/gov/defra/ffc/GitHubStatus.groovy (hunk 2 above)
Comment (fareedmohammed) [reply src/uk/gov/defra/ffc/GitHubStatus.groovy:56]: Agreed.

File: src/uk/gov/defra/ffc/GitHubStatus.groovy (hunk 3 above)
Comment (fareedmohammed) [reply src/uk/gov/defra/ffc/GitHubStatus.groovy:57]: Agreed.

File: vars/buildNodeJs.groovy (hunk 4 above)
Comment (fareedmohammed) [reply vars/buildNodeJs.groovy:85]: Good spot, forgot to remove from previous changes.

**Review by johnwatson484 — APPROVED** [review]
LGTM

**Pull request merged**